client/index.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import $ from 'jquery';
```

Comment:
Consider using something other than jQuery for ajax. there is a native mdn.io/fetch browser methods (tho they are limited) or to use https://www.npmjs.com/package/axios which is becoming the winner for HTTP clients.